hack/make/.mkwinres

```diff
@@ -9,14 +9,19 @@ quadVersionNum() {
 	fi
 }
 
+if [ ! -x "$(command -v go-winres)" ]; then
+	echo >&2 "go-winres not found, skipping manifesting binary"
+	exit 0
+fi
```

Comment:
dockerd.exe _has to_ be manifested or it will be subtly broken. Windows APIs will lie to dockerd unless it has a manifest declaring it supports Windows 10. Until we commit pregenerated .syso files containing stub manifests, a missing `go-winres` needs to be a fatal build error.